sdks/go/test/integration/integration.go

```diff
@@ -88,13 +88,17 @@ var flinkFilters = []string{
 }
 
 var samzaFilters = []string{
+	// XLang tests no supported by samza runner.
+	"TestXLang.*",
 	// TODO(BEAM-12608): Samza tests invalid encoding.
 	"TestReshuffle",
 	"TestReshuffleKV",
 	// The Samza runner does not support the TestStream primitive
 	"TestTestStream.*",
 	// The trigger tests uses TestStream
 	"TestTrigger.*",
+	// Samza doesn't yet support post job metrics, used by WordCount
+	"TestWordCount.*",
```

Comment:
Same, can we create a ticket ?

Reply:
I mean, we always can JIRA's are cheap. It's generally faster in cases like this to look for an existing one, and if not, create one, and then provide it than asking someone else to do it. In this case I've created it, because it's not worth the discussion time. BEAM-13006